spec/models/memberships/switch_stammsektion_spec.rb

```diff
@@ -45,19 +45,34 @@ def create_role(key, role, owner: person, **attrs)
       expect(errors).to eq ["Person muss Sac Mitglied sein"]
     end
 
-    it "is valid with membership in different section" do
-      create_role(:matterhorn_mitglieder, "Mitglied", start_on: 1.year.ago)
-      expect(switch).to be_valid
-    end
+    context "with membership in different section" do
+      let(:other_section) { groups(:matterhorn_mitglieder) }
+
+      it "is valid when active since before today" do
+        create_role(other_section, "Mitglied", start_on: 1.year.ago)
+        expect(switch).to be_valid
+      end
+
+      it "is valid when active since today" do
```

Comment:
Sollte hier ev. noch other section irgendwie in die description rein?

Reply:
ja das ist in der description des contextes drin :+1: